decoimpact/business/entities/rules/rolling_statistics_rule.py

```diff
@@ -36,59 +37,36 @@ def __init__(
         name: str,
         input_variable_names: List[str],
         operation_type: TimeOperationType,
-        operation_parameter: float = 0,
-        time_scale: str = "day",
-        period: float = 1,
-        output_variable_name: str = "output",
-        description: str = "",
     ):
-        super().__init__(name, input_variable_names, output_variable_name, description)
-        self._operation_type = operation_type
-        self._time_scale = time_scale.lower()
-        self._operation_parameter = operation_parameter
-        self._time_scale_mapping = {"hour": "H", "day": "D"}
```

Comment:
This seems a change in functionality: the variable period is replaced by a constant.